CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Extending the adopted spec, each change should have a link to its corresponding
 
 ## [Unreleased]
 
+### Added
+
+- Support for named cos images and option to specify cos family. [#37]
```

Comment:
The URL for #37 must be added to make this a valid link. Other URLs are at the bottom of the file.